dashboard/src/components/myProgress/MyProjectSkillsPage.vue

```diff
@@ -41,7 +41,7 @@ limitations under the License.
         theme: {
           disableSkillTreeBrand: true,
           disableBreadcrumb: true,
-          landingPageTitle: `PROJECT: ${this.$route.params.projectId}`,
+          landingPageTitle: `PROJECT: ${this.$route.params.name ? this.$route.params.name : this.$route.params.projectId}`,
```

Comment:
suggesting to load project name from the back-end if not available via route (in case of direct navigation use-case)